### DISCUSSION TITLE
git version on HPC

### DISCUSSION THREAD

**Discussion by johnyaku** [gitkraken/vscode-gitlens#2317] [Q&A]
GitLens works great locally on my laptop.

But I do a lot of work on a remove server (HPC), and there GitLab complains that the version of git is too old. The system-wide install of `git` is indeed fairly ancient (1.8.3.1) but I have default `conda` environment that is activated by `.bashrc`, and that contains `git` version 2.38.1.

However, GitLens doesn't appear to "see" the `git` in the active `conda` environment -- it still complains that the version is too old. Browsing the issues I found mention of a setting that points to the `git` executable, and discussion #1500 from May 2021 talks about a `git.path` setting but I can't find anything like that in the settings.

Suggestions?

**Comment by johnyaku** [comment]
Mysteriously, GitLens has stopped complaining about the version, and now seems to work. I didn't do anything in particular, except perhaps reconnect to the remote host. Maybe that did the trick ...

**Reply by fernandoroa** [reply]
mandatory step: https://stackoverflow.com/questions/55988854/alter-path-for-vs-code-remote-ssh

**Reply by johnyaku** [reply]
Thanks! I guess when I came back to it the next day VS Code Server restarted and everything started working. Now I know what to do to force a restart :)
